tests/test_utils.py

```diff
@@ -128,6 +128,11 @@ def test_pack_query_params_one(self):
             falcon.to_query_str({'things': ['a', 'b']}),
             '?things=a,b')
 
+        self.assertEqual(
+            falcon.to_query_str({'things': ['a', 'b', True, False]},
```

Comment:
I changed the test to: 

```python
self.assertEqual(
  falcon.to_query_str({'things': ['a', 'b', '', None, True, False, 0]},
    encode_lists_as_comma_delimited_string=False),
 '?things=a&things=b&things=&things=None&things=true&things=false&things=0')
```

I also noticed that `None` is not is converted to `'None'` since it if not special cased like `True` and `False`. Was there already a discussion about how to handle "NULL" values? I would prefer to translate `None` to the empty string.